doc/contributing/documentation_guide.md

```diff
@@ -217,57 +217,83 @@ For methods written in C, \RDoc cannot determine what arguments
 the method accepts, so those need to be documented using \RDoc directive
 [`:call-seq:`](rdoc-ref:RDoc::Markup@Method+arguments).
 
+For a singleton method, use the form:
+
+```c
+class_name.method_name(method_args) {|block_args| ... } -> return_type
+```
+
 Example:
 
 ```c
-*  call-seq:
-*    array.count -> integer
-*    array.count(obj) -> integer
-*    array.count {|element| ... } -> integer
+*  :call-seq:
```

Comment:
Do we encourage the form `:call-seq:` or `call-seq:`? They both seem to work but `call-seq:` seems to be much more common.

Reply:
Ruby uses `call-seq:` (1782 cases), not `:call-seq:` (1 case in string.c that should be fixed).

Reply:
Fixed (4 places).